patch-package
--- a/node_modules/babel-eslint/lib/patch-eslint-scope.js
+++ b/node_modules/babel-eslint/lib/patch-eslint-scope.js
@@ -233,6 +233,10 @@ function monkeypatch(modules) {
   // visit ClassPrivateProperty as a Property.
   referencer.prototype.ClassPrivateProperty = visitClassProperty;
 
+  // visit OptionalMemberExpression as a MemberExpression.
+  referencer.prototype.OptionalMemberExpression =
+    referencer.prototype.MemberExpression;
+
   // visit OptionalMemberExpression as a MemberExpression.
   referencer.prototype.OptionalMemberExpression =
     referencer.prototype.MemberExpression;
